frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/FieldListOption.tsx

```diff
@@ -79,6 +81,7 @@ export const DraggableBasicFieldListOption = ({
         <>
           <BasicFieldOption
             fieldType={fieldType}
+            isDisabled={isDisabled}
             {...props}
```

Comment:
/superNit: where the prop and arg name are the name, we can augment the props expansion:
```suggestion
            {isDisabled, ...props}
```

Reply:
probably not for prop spreading, this will fail hehe

Reply:
Oh, react spread doesn't follow standard object spread 😱 ? Apologies for the mis-information. I should have read the [docs](https://reactjs.org/docs/jsx-in-depth.html#spread-attributes) 😰😓.